applications/GeoMechanicsApplication/custom_workflows/dgeoflow.cpp

```diff
@@ -33,9 +33,7 @@ class GeoFlowApplyConstantScalarValueProcess : public Kratos::ApplyConstantScala
     {
     }
 
-    bool hasWaterPressure() const { return mvariable_name == "WATER_PRESSURE"; }
-
-    Kratos::ModelPart& GetModelPart() { return mr_model_part; }
```

Comment:
Can it be that this accidentally sneaked in?

Reply:
I think I don't get your point... Do you mean that the `GetModelPart` is now in the base `ApplyConstantScalarValueProcess`?

Reply:
Yes, as should be

Reply:
Done

Reply:
Sorry https://github.com/KratosMultiphysics/Kratos/actions/runs/8387073733/job/22968574304, I will revert it partially